### DISCUSSION TITLE
Add on Forks page count of ahead and below commits

### DISCUSSION THREAD

**Discussion by mathewsun** [community/community#27766] [Other Feature Feedback, Questions, & Ideas]
Add on Forks page count of ahead and below commits.
Making sorting on this params.

**Comment by jfullstackdev** [comment]
what do you mean by that ?

**Reply by mathewsun** [reply]
> what do you mean by that ?

![image](https://user-images.githubusercontent.com/2837780/186046706-5493d8fd-d0b1-4ff8-bd9a-5bed82327518.png)

**Reply by mathewsun** [reply]
on forks page show how many commits ahead and behind on every fork of current repo

**Reply by mathewsun** [reply]
![image](https://user-images.githubusercontent.com/2837780/186047739-89674cbc-c5d0-4362-9ad2-600a2ee552e6.png)

**Reply by mathewsun** [reply]
like this

**Comment by mathewsun** [comment]
becouse you need to open every fork to find new functionality

**Comment by mathewsun** [comment]
ahead commits are more interested
